StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/StandardTextDiagnosticBase.cs

```diff
@@ -193,7 +193,7 @@ private static bool SeeTagIsCorrect(SyntaxNodeAnalysisContext context, XElement
             }
 
             INamedTypeSymbol expectedSymbol = semanticModel.GetDeclaredSymbol(constructorDeclarationSyntax.Parent, context.CancellationToken) as INamedTypeSymbol;
-            return actualSymbol.OriginalDefinition == expectedSymbol;
+            return actualSymbol.OriginalDefinition.Equals(expectedSymbol);
```

Comment:
@sharwell Do you happen to know whether `OriginalDefinition` can return null?

Reply:
I am not certain